Replace 'nullptr' by NULL in order to allow compiling in C++98 mode.
See also: https://bugs.gentoo.org/show_bug.cgi?id=599618

--- a/src/af/gr/gtk/gr_UnixCairoGraphics.cpp
+++ b/src/af/gr/gtk/gr_UnixCairoGraphics.cpp
@@ -180,7 +180,7 @@
 	if (m_styleBg) {
 		g_object_unref(m_styleBg);
 	}
-	m_styleBg = XAP_GtkStyle_get_style(nullptr, "GtkButton"); // "button"
+	m_styleBg = XAP_GtkStyle_get_style(NULL, "GtkButton"); // "button"
 	// guess colours
 	// WHITE
 	GdkRGBA rgba2;
@@ -627,7 +627,7 @@
 		_setProps();
 		cairo_save (m_cr);
 
-		GtkStyleContext *context = nullptr;
+		GtkStyleContext *context = NULL;
 		switch(c) {
 		case GR_Graphics::CLR3D_Background:
 			context = m_styleBg;
--- a/src/wp/ap/xp/ap_Dialog_Spell.cpp
+++ b/src/wp/ap/xp/ap_Dialog_Spell.cpp
@@ -251,7 +251,7 @@
 					UT_return_val_if_fail (m_Suggestions, false);
 
 					// get suggestions from spelling engine
-					const UT_GenericVector<UT_UCSChar*> *cpvEngineSuggestions = nullptr;
+					const UT_GenericVector<UT_UCSChar*> *cpvEngineSuggestions = NULL;
 
 					if (checker->checkWord(m_pWord, m_iWordLength) == SpellChecker::LOOKUP_FAILED)
 					{
